src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const tHead = document.querySelector('thead');
+const tBody = document.querySelector('tbody');
+const rows = [...tBody.children];
+let reverse = false;
+let value = '';
+
+const handleEvent = (e) => {
+  const header = [...document.querySelectorAll('th')];
+  const num = header.findIndex(
+    item => item.textContent === e.target.textContent
+  );
+
+  if (value === e.target.textContent) {
+    reverse = !reverse;
+  } else {
+    reverse = false;
+  }
```

Comment:
You also could use a ternary operator here 
```suggestion
  reverse = value === e.target.textContent ? !reverse : false;
```